misc/prometheus/prometheus.yml

```diff
@@ -0,0 +1,18 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+global:
+  scrape_interval: 15s
+scrape_configs:
+  - job_name: "environmentd"
+    static_configs:
+    - targets: ["host.docker.internal:6878"]
```

Comment:
This won't work on Linux without additional config. See https://stackoverflow.com/a/24326540. Recommend just passing `--network=host` in the `docker run` instructions. Then you won't need to do anything special to expose port 9090, and Materialize's internal HTTP port will be available at `localhost:6878`.

Reply:
"The host networking driver only works on Linux hosts, and is not supported on Docker Desktop for Mac, Docker Desktop for Windows, or Docker EE for Windows Server."

I think ill just go with: `--add-host host.docker.internal:host-gateway` in the docs for linux?